### PR TITLE
Resolved Bug 467, 482 and 485 : Resolved Issues on Counter, Collapse, Button and Card Element Issues

### LINKED ISSUE #467: Elements >Counter > Selecting light theme & color Variant light outline is not display.
**Describe the bug**
Selecting light theme & color Variant light outline is not display.
Selecting dark theme & color Variant dark+,- icon is not display
Element Collapse Click on toggle element open popup alignment issue.

**To Reproduce**
1. Go to 'Elements'
2.Click on 'Counter '
3.Click on 'Default'

**Screenshots**
image.

image

image

**Desktop (please complete the following information):**
 -OS: [Windows 10 Enterprise]
-Browser [ edge]
-Version [ 125.0.2535.92]

**Smartphone (please complete the following information):NA**

**Additional context**

### LINKED ISSUE #482: Storybook > Elements >Button> Default >Selecting dark theme colorVariant light button text is not display.
**Describe the bug**
Selecting dark theme colorVariant light button text is not display.
Check all stories Default, Disable, With Icon, Tooltip, Text With Icon.

**To Reproduce**
1. Go to 'Elements '
2. Click on 'Button'
3. Click on 'Default '

**Expected behavior**
Selecting dark theme colorVariant light button text, icon is visible.

**Screenshots**
image.

image

image

image

image

**Desktop (please complete the following information):**
-OS: [Windows 10 Enterprise]
-Browser [ edge]
-Version [ 125.0.2535.92]

**Smartphone (please complete the following information):NA**

**Additional context**

### LINKED ISSUE #485: Storybook > Elements >Card> Avatar >Unable to see image.
**Describe the bug**
Card > Default> Selecting dark theme color variant light button text is not visible.
Card > Card with Image > Selecting dark theme color variant light button text is not visible.
Card >Avatar > Selecting dark theme color variant light button text &Image is not visible.
Card >With Centered Avatar > Selecting dark theme color variant light button text &Image is not visible.

**To Reproduce**
1. Go to 'Elements'
2. Click on 'Card'
3.Click on 'Avatar'

**Screenshots**
image.

image

image

**Desktop (please complete the following information):**
-OS: [Windows 10 Enterprise]
-Browser [ edge]
-Version [ 125.0.2535.92]

**Smartphone (please complete the following information):NA**

**Additional context**

### PR DESCRIPTION
**Pull request by AkshayPotphode**
-  **Counter**
-  **Collapse**
-  **Button**
-  **Card**

> Make the changes to rds-button.css, rds-card.stories.tsx, rds-card.tsx, rds-collapse.tsx & rds-counter.css file.
> Resolve issue on light color varient outline is not visible & in dark theme with dark color varient +,- icons is not visible in counter element.
> Resolved spacing issue on Toggle element.
> Resolved issue on selecting dark theme with color varient light button text is not visible on all stories in button element.
> Resolved issue on selecting dark theme with light color varient button text is not visible on all stories & Avatar and with centered avatar, Avatar profile is not visible.

## Screenshots :
 
image

image

image

image

image

image

 
## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: raaghu-elements/src/rds-button/rds-button.css [hunk 1]
```diff
@@ -19,3 +19,11 @@
 .alignClass {
   top: -1px;
 }
+
+.theme-dark .btn-light {
```
Comment (suraj-khupse) [inline raaghu-elements/src/rds-button/rds-button.css:23]: please do not used hard code color variable

File: raaghu-elements/src/rds-collapse/rds-collapse.tsx [hunk 2]
```diff
@@ -21,14 +21,14 @@ const RdsCollapse = (props: RdsCollapseProps) => {
             <div>
                 <button
                     type="button"
-                    className="btn btn-primary btn-sm"
+                    className="btn btn-primary btn-sm ms-1"
```
Comment (suraj-khupse) [inline raaghu-elements/src/rds-collapse/rds-collapse.tsx:24]: Check with raaghu application as well , add screenshot

File: raaghu-elements/src/rds-counter/rds-counter.css [hunk 3]
```diff
@@ -4,3 +4,16 @@
 button {
   background-color: red;
 }
+
+.btn-light {
+  border-color: #E6E7EB;
+}
```
Comment (suraj-khupse) [inline raaghu-elements/src/rds-counter/rds-counter.css:10]: remove css from element

**Review by suraj-khupse — CHANGES_REQUESTED** [review]

**Comment by AkshayPotphode** [comment]
Done

File: raaghu-elements/src/rds-button/rds-button.css (hunk 1 above)
Comment (AkshayPotphode) [reply raaghu-elements/src/rds-button/rds-button.css:23]: No change

File: raaghu-elements/src/rds-counter/rds-counter.css (hunk 3 above)
Comment (AkshayPotphode) [reply raaghu-elements/src/rds-counter/rds-counter.css:10]: Added css in counter.css

File: raaghu-elements/src/rds-collapse/rds-collapse.tsx (hunk 2 above)
Comment (AkshayPotphode) [reply raaghu-elements/src/rds-collapse/rds-collapse.tsx:24]: Not required

**Review by suraj-khupse — APPROVED** [review]

**Review by suraj-khupse — APPROVED** [review]

**Pull request merged**